src/mito2/src/memtable.rs

```diff
@@ -69,6 +69,8 @@ pub struct MemtableStats {
     /// The time range that this memtable contains. It is None if
     /// and only if the memtable is empty.
     time_range: Option<(Timestamp, Timestamp)>,
+    /// Total rows in memtable
+    num_rows: usize,
```

Comment:
We should document that it also contains the deleted rows.

Reply:
fixed in 92e5295